src/chat/panes/left-menu/template.pug

```diff
@@ -1,8 +1,7 @@
 h1 Menu
 div.buttons
-    div.button1(ng-click = "$ctrl.setItem('newShoppingCard')") Yeni Alışveriş Listesi
-    div.button1(ng-class = "{'archiveScreen': $ctrl.uptodateScreen}" ng-click = "$ctrl.setItem('Uptodate')") Mevcut Listeler
-    div.button1(ng-class = "{'archiveScreen': $ctrl.archiveScreen}"  ng-click = "$ctrl.setItem('Archive')") Arşiv
-    div.button1(ng-class = "{'archiveScreen': $ctrl.recycleBinScreen}"  ng-click = "$ctrl.setItem('Delete')") Çöp Kutusu
-    div.button1(ng-class = "{'archiveScreen': $ctrl.userScreen}"  ng-click = "$ctrl.setItem('User')") Kişiler
-
+    div.button1(ng-click = "$ctrl.newShoppingCard()") Yeni Alışveriş Listesi
+    div.button1(ng-class = "{'archiveScreen': $ctrl.screenType == $ctrl.screenTypes.UPTODATE}" ng-click = "$ctrl.setItem($ctrl.screenTypes.UPTODATE)") Mevcut Listeler
+    div.button1(ng-class = "{'archiveScreen': $ctrl.screenType == $ctrl.screenTypes.ARCHIEVE}"  ng-click = "$ctrl.setItem($ctrl.screenTypes.ARCHIEVE)") Arşiv
+    div.button1(ng-class = "{'archiveScreen': $ctrl.screenType == $ctrl.screenTypes.RECYCLEBIN}"  ng-click = "$ctrl.setItem($ctrl.screenTypes.RECYCLEBIN)") Recycle Bin

```

Comment:
Are you sure about the `(ng-class = "{'archiveScreen': $ctrl.screenType == $ctrl.screenTypes.RECYCLEBIN}"` declaration here? Why is it always the class `archiveScreen` regardless of the screen type? What does this class do?